tools/run_pylint.sh

```diff
@@ -16,4 +16,10 @@
 # Ensure we are at the project root directory
 cd $(readlink -f $(dirname $0))/..
 
-pylint --score=n `git ls-files --deduplicate --exclude-standard --full-name '*.py'`
+if [ -f "py_virtual_env/bin/pylint" ]; then
```

Comment:
When may this not exist?

Reply:
In our GitHub workflow for Python style. I was considering rewriting the workflow, but this seemed easier.

Reply:
I go with option 2, more elegant.

I'd have to also install `yapf` into our venv in the workflow if it would use setup / venv. And exclude more temporary directories. And it would take longer with the Rust setup that we don't need for Python style.